src/core/server/http/types.ts

```diff
@@ -276,8 +285,33 @@ export interface InternalHttpServiceSetup
 
 /** @public */
 export interface HttpServiceStart {
-  /** Indicates if http server is listening on a given port */
-  isListening: (port: number) => boolean;
+  /**
+   * Access or manipulate the Kibana base path
+   * See {@link IBasePath}.
+   */
+  basePath: IBasePath;
+
+  /**
+   * Auth status.
+   * See {@link HttpAuth}
+   */
+  auth: HttpAuth;
+
+  /**
+   * Flag showing whether a server was configured to use TLS connection.
+   */
+  isTlsEnabled: boolean;
+
+  /**
+   * Provides common {@link HttpServerInfo | information} about the running http server.
+   */
+  getServerInfo: () => HttpServerInfo;
+}
+
+/** @internal */
+export interface InternalHttpServiceStart extends HttpServiceStart {
+  /** Indicates if the http server is listening on the configured port */
+  isListening: () => boolean;
```

Comment:
Previous signature was `isListening: (port: number) => boolean`, but the actual implementation was `() => boolean`.

Reply:
It was leftover of node synchronization mechanism logic.